### ISSUE TITLE
primevue-plugin error after installation

### ISSUE THREAD

**Issue by richcolemantx** [primefaces/primevue-tailwind#61]
After doing the install steps from the website:

> npx nuxi module add @nuxtjs/tailwindcss
> npx tailwindcss init
> npm install primevue
> npm install --save-dev nuxt-primevue

```
export default defineNuxtConfig({
    modules: [
        'nuxt-primevue'
    ],
    primevue: {
        unstyled: true,
        importPT: { from: path.resolve(__dirname, './presets/lara/') }      //import and apply preset   
    }
})
```

I'm getting the following error after `npm run dev`:

> 500
> [vite-node] [plugin:unctx:transform] [VITE_ERROR] virtual:nuxt:E:/GitHub/my-account/my-repo/.nuxt/primevue-plugin.mjs:14:65
> 
> 12 |  import FocusTrap from 'primevue/focustrap';
> 13 |  import AnimateOnScroll from 'primevue/animateonscroll';
> 14 |  import undefined from 'E:\GitHub\my-account\my-repo\ui-presets\my-site';
>    |                                                                   ^
> 15 |  
> 16 |  
> 
> at virtual:nuxt:E:/GitHub/my-account/my-repo/.nuxt/primevue-plugin.mjs:14:65

When I build, I also get this error:

> ERROR  [unctx:transform] Bad character escape sequence (14:65) 
> file: virtual:nuxt:E:/GitHub/my-account/my-repo/.nuxt/primevue-plugin.mjs:14:65

If I comment out the importPT line, the site loads fine and renders the control but doesn't style it at all.

`<Button label="Submit" />`

![image](https://github.com/primefaces/primevue-tailwind/assets/91837210/7e96fd9a-4e1c-4723-8dcd-29ff59e7ec70)

**Comment by richcolemantx** [comment]
I changed my folder names for presets to `./presets/default/` and I got past that error but I'm on to another error after `npm run dev`

`y-accountmy-repopresetsdefault" from "virtual:nuxt:E:\GitHub\my-account\my-repo\.nuxt\primevue-plugin.mjs". Does the file exist?`

Note the missing letter `m` in the name `y-accountmy-repopresetsdefault"` and the rogue double quote.

**Comment by richcolemantx** [comment]
I tried again with a fresh install with a basic startup nuxt project (nuxt welcome page loaded up fine), added tailwind prereqs (worked fine here with tailwind Hello World css), and then I added PrimeVue and got this error:

![image](https://github.com/primefaces/primevue-tailwind/assets/91837210/eef1b34d-c585-4c0a-90b8-8c6dc491eb2b)

Nuxt 3.9.1 with Nitro 2.8.1                                                                                                                                             7:50:38 PM
[get-port] Unable to find an available port (tried 3000-3100 on host "localhost"). Using alternative port 3001                                                          7:50:39 PM
                                                                                                                                                                        7:50:39 PM
  ➜ Local:    http://localhost:3001/
  ➜ Network:  use --host to expose

  ➜ DevTools: press Shift + Alt + D in the browser (v1.0.6)                                                                                                             7:50:42 PM

ℹ Compiled plugins/server.mjs in 687.14ms                                                                                                                              7:50:44 PM
ℹ Compiled plugins/client.mjs in 701.7ms                                                                                                                               7:50:44 PM
ℹ Compiled types/plugins.d.ts in 703.64ms                                                                                                                              7:50:44 PM

eposprimetestpresetslara" from "virtual:nuxt:C:\Users\rich\source\repos\primetest\.nuxt\primevue-plugin.mjs". Does the file exist?

ℹ Vite client warmed up in 3030ms                                                                                                                                      7:50:48 PM
ℹ Vite server warmed up in 3165ms                                                                                                                                      7:50:49 PM
✔ Nitro built in 2785 ms
